2021/02/22 03:49:30 : customers_errors 1: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 1034, in convert_customer_export
    state = get_row_from_list_by_field(customers_ext['data']['zones'], 'zone_code', state_id)
KeyError: 'zones'

2021/02/22 03:49:30 : customers_errors 3: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 1034, in convert_customer_export
    state = get_row_from_list_by_field(customers_ext['data']['zones'], 'zone_code', state_id)
KeyError: 'zones'

2021/02/22 03:49:40 : order id 5 import failed. Error: Exceeded  API rate limit, please try again in a minute. Upgrade to a paid account to remove this limit.
2021/02/22 03:49:43 : order id 6 import failed. Error: Exceeded  API rate limit, please try again in a minute. Upgrade to a paid account to remove this limit.
2021/02/22 03:49:46 : order id 7 import failed. Error: Exceeded  API rate limit, please try again in a minute. Upgrade to a paid account to remove this limit.
2021/02/22 03:49:49 : order id 8 import failed. Error: Exceeded  API rate limit, please try again in a minute. Upgrade to a paid account to remove this limit.
